lib/jsonapi/serializer.ex

```diff
@@ -38,6 +38,7 @@ defmodule JSONAPI.Serializer do
       id: view.id(data),
       type: view.type(),
       attributes: view.attributes(data, conn),
+      meta: view.meta(data, conn),
```

Comment:
I think we should only add meta if view.meta() is not nil? The spec doesn't state if we should send down empty maps or not but if someone isn't setting meta perhaps they'd prefer it not be sent down?

Subsequently we should return nil here. https://github.com/jeregrine/jsonapi/pull/62/files#diff-c44c9da7e2c91d6f0643f4ef73ab636cR88